topi/python/topi/x86/conv2d_avx_1x1.py

```diff
@@ -146,6 +152,8 @@ def _schedule_conv(s, data, data_pad, data_vec, kernel, kernel_vec, conv_out, ou
     s[O].reorder(oc_chunk, oh_outer, ow_outer, oh_inner, ow_inner, oc_block)
 
     parallel_axis = s[O].fuse(oc_chunk, oh_outer)
+    if sch.parallel_chunk > 0:
```

Comment:
Similar as the comment above, we may not need an extra parameter here but just a function to generate parallel_chunk.